### DISCUSSION TITLE
Edit command not work.

### DISCUSSION THREAD

**Discussion by OriLiMu** [sxyazi/yazi#1240] [1. Q&A]
Here is my config. But nothing happens when I press <kbd>e</kbd> on item. 
```
-- yazi.toml
[opener]
edit = [
	{ run = '$EDITOR "$@"', block = true,  for = "unix" },
	{ run = 'code "%*"',    orphan = true, for = "windows" },
]

-- keymap.toml
	{ on = [ "e" ],         run = "edit",                                                 desc = "Open the selected files" },

```

> 
> Yazi
>     Version: 0.2.5 (c64530b 2024-06-26)
>     Debug  : false
>     OS     : linux-x86_64 (unix)
> 
> Ya
>     Version: 0.2.5
> 
> Emulator
>     Emulator.via_env: ("xterm-kitty", "")
>     Emulator.via_csi: Ok(Unknown([Sixel]))
>     Emulator.detect : Kitty
> 
> Adapter
>     Adapter.matches: X11
> 
> Desktop
>     XDG_SESSION_TYPE: Some("x11")
>     WAYLAND_DISPLAY : None
>     DISPLAY         : Some(":0")
> 
> SSH
>     shared.in_ssh_connection: false
> 
> WSL
>     /proc/sys/fs/binfmt_misc/WSLInterop: false
> 
> Variables
>     SHELL              : Some("/usr/bin/fish")
>     EDITOR             : Some("nvim")
>     ZELLIJ_SESSION_NAME: Some("拼搏与梦想")
>     YAZI_FILE_ONE      : None
>     YAZI_CONFIG_HOME   : None
> 
> Text Opener
>     default: Some(Opener { run: "$EDITOR \"$@\"", block: true, orphan: false, desc: "$EDITOR", for_: None, spread: true })
>     block  : Some(Opener { run: "$EDITOR \"$@\"", block: true, orphan: false, desc: "$EDITOR", for_: None, spread: true })
> 
> tmux
>     TMUX   : false
>     Version: No such file or directory (os error 2)
> 
> Dependencies
>     file             : 5.45
>     ueberzugpp       : 2.9.6
>     ffmpegthumbnailer: 2.2.2
>     magick           : 7.1.1-33
>     fzf              : 0.52.0
>     fd               : 10.1.0
>     rg               : 14.1.0
>     chafa            : 1.14.0
>     zoxide           : 0.9.4
>     unar             : 1.10.7
>     jq               : No such file or directory (os error 2)
> 
> 
> --------------------------------------------------
> When reporting a bug, please also upload the `yazi.log` log file - only upload the most recent content by time.
> You can find it in the "/home/lizhe/.local/state/yazi" directory.
> 
>

**Comment by sxyazi** [comment]
There's no such command called `edit`, see full commands at https://yazi-rs.github.io/docs/configuration/keymap

**Reply by OriLiMu** [reply]
hi why this command just open nvim not edit the file . Neither work
```toml
{ on = [ "e" ],  run = "shell --block --confirm nvim $0",   desc = "Open the selected files" },
{ on = [ "e" ],  run = "shell --block --confirm nvim run $0", desc = "Open the selected files" },
```
